### DISCUSSION TITLE
Turning StreetComplete into a maps app?

### DISCUSSION THREAD

**Discussion by ThatOneCalculator** [streetcomplete/StreetComplete#4263] [Ideas]
I just downloaded StreetComplete and this is by far the cleanest implementation of OSM I've seen on mobile, and I love what this project is doing? Are there any plans to maybe use this interface for a more traditional maps app?

**Comment by matkoniecz** [comment]
no, but it is an open source project so you can reuse code (though not the name)

also, I am sometimes using it in this way

**Comment by BalooUriza** [comment]
I would love to see a streetcomplete plugin for Osmand, but that's more of
a Osmand issue than SC issue.

**Comment by legofahrrad** [comment]
You can see just the map on https://streetcomplete.github.io/streetcomplete-mapstyle/ – this works on every OS, not just Android ;)

**Reply by ThatOneCalculator** [reply]
Yea but that doesn't work as a navigation app lol

**Comment by mnalis** [comment]
If you want navigation with non-cluttered UI, you may wish to try [Organic Maps](https://wiki.openstreetmap.org/wiki/Organic_Maps). 
Although [OsmAnd](https://wiki.openstreetmap.org/wiki/OsmAnd) can also use dozens of different map styles and its UI can be trimmed down via extensive options it offers.

**Reply by ThatOneCalculator** [reply]
I currently use OrganicMaps (been using it for about a month), which is pretty nice.
